com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyLanguageServerImpl.java

```diff
@@ -66,21 +67,23 @@
   private WorkspaceService workspaceService;
   private CopybookService copybookService;
   private Provider<LanguageClient> clientProvider;
-
   private SettingsProvider settingsProvider;
+  private CopybookDependencyService copybookDependencyService;
 
   @Inject
   MyLanguageServerImpl(
       CopybookService copybookService,
       TextDocumentService textService,
       WorkspaceService workspaceService,
       Provider<LanguageClient> clientProvider,
-      SettingsProvider settingsProvider) {
+      SettingsProvider settingsProvider,
+      CopybookDependencyService copybookDependencyService) {
```

Comment:
```copybookDependencyService``` is unused within the class

Reply:
by calling it here I initialized it in order to subscribe it to the databus